### PR TITLE
feat: center div

### PR DESCRIPTION
**Pull request by tranmarta**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
The goal of the task is to center a `div` with the `.box` class both horizontally and vertically within a `div` with the `.arena` class. The styles should be written in the `styles.css` file.

### Code Changes
The code changes were made in the `styles.css` file. The `display: center;` property was added to both the `.box` and `.arena` classes.

### Review
The changes do not meet the task requirements. The `display: center;` property does not exist in CSS, so it will not have any effect on the positioning of the `.box` within the `.arena`. 

To center a block element both vertically and horizontally within its parent, you can use a combination of different CSS properties. One common method is to use `display: flex;`, `justify-content: center;`, and `align-items: center;` on the parent element (in this case, `.arena`). 

Here is an example of how you could modify the `.arena` class to achieve this:

```css
.arena {
    display: flex;
    justify-content: center;
    align-items: center;
    border: 1px dotted olive;
    height: 80vh;
}
```

This will center any child elements of `.arena` both horizontally and vertically. 

Please update your code accordingly to meet the task requirements.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
Center the `.box` horizontally and vertically in `.arena`.

### Requirements
1. Styles should be written in `styles.css` file.
2. `div` with `.box` class should be in the middle of `div` with class `.arena`.

### Code Changes Review

The changes have been made in the `styles.css` file as per the first requirement.

However, the changes do not fulfill the second requirement. The `display: center;` property is not valid in CSS. To center an element both vertically and horizontally, you could use a combination of `display: flex;`, `justify-content: center;`, and `align-items: center;`.

For example, the `.arena` class should look like this:

```css
.arena {
    border: 1px dotted olive;
    height: 80vh;
    display: flex;
    justify-content: center;
    align-items: center;
}
```

This will center any child elements of `.arena`, including `.box`, both horizontally and vertically.

The `.box` class does not need any changes for centering as it will be centered by its parent `.arena`. The added line `display: center;` should be removed.

In conclusion, the code changes do not cover the task requirements. The code needs to be corrected as suggested above to fulfill the task requirements.
